### PR TITLE
Make it possible to configure if icons are shown.

### PR DESCRIPTION
**Pull request by tow8ie**
This change allows to disable the display of the “X” and the “checkmark” icon with a new `showIcons` property which is set to `true` by default so that it mimics react-toggle’s old behavior in the standard case.

### CODE REVIEW

**Review by ahstro — CHANGES_REQUESTED** [review]
I like this feature; if you resolve the merge conflicts, I'll test it out and see if it works as intended :)

Thank you!

File: index.es6.js [hunk 1]
```diff
@@ -73,10 +80,10 @@ export default React.createClass({
       <div className={classes} onClick={this.handleClick}>
         <div className="react-toggle-track">
           <div className="react-toggle-track-check">
-            <Check />
+            {this.props.showIcons ? <Check /> : ''}
           </div>
           <div className="react-toggle-track-x">
-            <X />
+            {this.props.showIcons ? <X /> : ''}
```
Comment (ahstro) [inline index.es6.js:86]: Could you use `null` intead of empty string literals for these? IMO, it makes it clearer what we're trying to do. :)

Comment (tow8ie) [reply index.es6.js:86]: No problem.

**Comment by tow8ie** [comment]
I’m not able to work on this PR before Tuesday. But I’ll prepare it for a merge as soon as I have time to do so.

**Comment by ahstro** [comment]
No problem, no rush :)

**Comment by ahstro** [comment]
Moving milestone from this PR to the issue where it is being discussed (#56) :)

**Comment by ahstro** [comment]
Implemented by #70. Sorry I took a PR that was opened after this, but you didn't respond :/

**Pull request closed without merging**